Tiltfile

```diff
@@ -29,35 +33,36 @@ ignore=[
   'packages/'
 ]
 
+if cfg.get('defaultRegistry') != None:
+  default_registry(cfg.get('defaultRegistry'))
+
 local_resource('build charts',
   deps='packages/**/templates',
-  cmd='mage charts',
+  cmd='go run ./dagger --charts.git.export',
   ignore=ignore,
 )
 
-k8s_yaml(helm('./charts/opni-crd/'+version,
-  name='opni-crd',
+helm_resource('opni-crd', './charts/opni-crd/'+version,
   namespace='opni',
-), allow_duplicates=True)
-
-if cfg.get('valuesPath') != None:
-  k8s_yaml(helm('./charts/opni/'+version,
-    name='opni',
-    namespace='opni',
-    values=cfg.get('valuesPath')
-  ), allow_duplicates=True)
-else:
-  k8s_yaml(helm('./charts/opni/'+version,
-    name='opni',
-    namespace='opni',
-    set=cfg.get('opniChartValues')
-  ), allow_duplicates=True)
+  release_name='opni-crd',
+  resource_deps=['build charts'],
+  auto_init=False
+)
+# workaround for https://github.com/tilt-dev/tilt/issues/6058
+k8s_resource('opni-crd', auto_init=True, pod_readiness='ignore')
 
-if cfg.get('defaultRegistry') != None:
-  default_registry(cfg.get('defaultRegistry'))
+helm_resource('opni', './charts/opni/'+version,
+  namespace='opni',
+  release_name='opni',
+  resource_deps=['build charts', 'opni-crd'],
+  image_selector="rancher/opni",
+  image_deps=['rancher/opni'],
+  image_keys=[("image.repository", "image.tag")],
```

Comment:
keep compatibility with `cfg.get('opniChartValues')` and `values=cfg.get('valuesPath')` by using `flags` option. 

From the docs :
- Pass values as flags=['--set=key=value'].
- Pass a values file as flags=['--values=./path/to/values.yaml'].